rms-frontend/apps/s3/src/app/app.module.ts

```diff
@@ -5,6 +5,7 @@ import { CoreModule } from '@rms-frontend/core';
 import { AppComponent } from './app.component';
 import { NgxsModule } from '@ngxs/store';
 import { NgxsReduxDevtoolsPluginModule } from '@ngxs/devtools-plugin';
+import { NgxsStoragePluginModule } from '@ngxs/storage-plugin';
```

Comment:
This was all the config needed to make state save and pull from localstorage.....

Reply:
lol nice